docs/_attic/glossary_entries.py

```diff
@@ -22,6 +22,12 @@
 AGC = GlossEntry("AGC",
 	acronym_full="[Amazon Genomics CLI]")
 
+AbsolutePath = GlossEntry("absolute path",
+	definition="A path that starts with the character ``/`` and contains the full set of directories necessary to resolve a file, starting from the root directory. For example: ``/Dockstore.cwl`` or ``/bin/sh``",
```

Comment:
> from the root directory

I'm not sure how to work it in, and especially here where it's a generic definition, but the absolute path in the context of the bug is "relative" :) to the repository's root, not the (local) file system root.